file_formats/ie_formats/itm_v1.htm

```diff
@@ -2,6 +2,7 @@
 title: "ITM file format"
 regenerate: true
 ---
+    {% include relurl.html %}
```

Comment:
Aren't these changes noops? None of the links have been touched.

Reply:
They enable relurl to be work in the files. Without it, contributors following the guideline will find that it results in broken links.
I updated one link to check that it works.